coverage.txt

```diff
@@ -1,13 +1,13 @@
 Name                        Stmts   Miss  Cover   Missing
 ---------------------------------------------------------
-symr/benchmark.py             147      7    95%   56, 61, 68, 231, 236, 263-264
+symr/benchmark.py             148      7    95%   56, 61, 68, 232, 237, 264, 269
 symr/fake_sr.py                83      2    98%   72-73
-symr/helpers.py                 7      0   100%
-symr/metrics.py                80      0   100%
+symr/helpers.py                40     22    45%   24, 38-77
+symr/metrics.py                82      0   100%
 symr/nsrts_wrapper.py         167     15    91%   28, 39, 63-64, 215, 269, 277-285, 290-291, 346
 symr/pysr_wrapper.py           55     10    82%   23, 44-45, 50-52, 80-87
 symr/symformer_wrapper.py      61     11    82%   23-24, 53-54, 81-89, 94
 symr/symgpt_wrapper.py        143     20    86%   27, 57, 60, 79-80, 88-94, 142-150, 240
 symr/wrappers.py               12      0   100%
 ---------------------------------------------------------
-TOTAL                         755     65    91%
+TOTAL                         791     87    89%
```

Comment:
✓
Note that getting those missing 22 lines in symr/helpers.py shouldn't be very hard (it's the r2 curve plotting helper)